www/source/docs/blueprints.md

```diff
@@ -0,0 +1,58 @@
+# Blueprints
+Blueprints are default implementations of the build and runtime phases for your application. They also provide some default health check hooks where appropriate to ensure your application functioning reliably. Customized Blueprints can be created to facilitate reusability of common patterns in your organization for developing, building, and running your applications.
+
+## Getting Started
+### New Projects
+Simply run `hab plan init` to leverage blueprints. Habitat will look in your application code for known project types and the appropriate `pkg_blueprint`.
+
+If you would like to specify the desired blueprint to use, you can run `hab plan init java8-maven-tomcat`
+
+### Existing Plans
+
+To begin using Blueprints, you will need to add the appropriate `pkg_blueprints` 
+
+```bash
+pkg_name="my_java_app"
+....
+pkg_blueprints=("core/blueprints/java8-maven-tomcat")
+               ^     ^^^^^^^^^^^                    ^
+               |______|_______|_____________________|
+                 Arrays and additional namespace TBD -- just random thoughts at the moment.
+```
+* add `auto` package to `build_deps`
+* set toolchain variable
+
+#### Optional Variables
+* `pkg_src` - now accepts URI notation. Each blueprint contains a default URI which is relevant to the type of blueprint being used.
+*  `pkg_blueprint_opts` - Options which override some default runtime parameters. Eg: `--env DEV`. This operates similar to `pkg_svc_run` as the result of using this option will intelligently generate a hook file for you. However, you also can create the [hook file](#Run-Hooks) directly if you need to have more control over the resulting output.
+
+### Available Blueprints
```

Comment:
This looks like a great list for our initial offerings even though I know you said it's a mockup ;). We should get some feedback from @tashimi on what she think, too.

@fnichol and I spoke about tackling `go`  as one of the first, I think that could be an easy home run